pwndbg/regs.py

```diff
@@ -280,7 +280,7 @@ def __getattr__(self, attr):
 
             value = int(value)
             return value & pwndbg.arch.ptrmask
-        except (ValueError, gdb.error):
+        except (ValueError, AttributeError, gdb.error) as e:
```

Comment:
whenever some code accesses the regs.pc while its not yet available, i remove it for not and we handle it later. too much stuff